examples/vnet/doca_gw.c

```diff
@@ -81,15 +82,15 @@ struct doca_gw_port * doca_gw_port_start(struct doca_gw_port_cfg *cfg, struct do
 		//TODO: need to parse devargs
 		DOCA_LOG_INFO("new doca port type:dpdk port id:%s", cfg->devargs);
 		port = doca_gw_dpdk_port_start(cfg, err);
-		if (port)
-			port->port_id = atoi(cfg->devargs);
```

Comment:
this is the way we take the user information. note that in the futrue devargs might have additional information such as hair-pin queues...etc.

Reply:
done